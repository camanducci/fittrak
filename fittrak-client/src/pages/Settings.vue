<template>
  <v-container>
    <SettingsDataContainer>
      <template slot-scope="slotProps"> 
        <UserSettings :user="slotProps.user" />
      </template>
    </SettingsDataContainer>
  </v-container>
</template>

<script>
import SettingsDataContainer from "@/components/settings/SettingsDataContainer";
import UserSettings from "@/components/settings/UserSettings";

export default {
  name: "SettingsPage",

  components: {
    SettingsDataContainer,
    UserSettings
  }
};
</script>
