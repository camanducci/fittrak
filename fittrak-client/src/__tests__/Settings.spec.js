import { shallowMount } from "@vue/test-utils";
import Settings from "@/pages/Settings";

describe("Settings", () => {
  test("Settings page renders", () => {
    const app = shallowMount(Settings);
    expect(app.html()).toMatchSnapshot();
  });
});
