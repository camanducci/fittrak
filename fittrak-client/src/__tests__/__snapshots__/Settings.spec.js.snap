// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings Settings page renders 1`] = `
<v-container-stub tag="div">
  <v-layout-stub tag="div" row="" wrap="">
    <v-flex-stub tag="div" xs12="">
      <h2 class="display-1">Settings</h2>
      <v-divider-stub></v-divider-stub>
      <p>Nothing to see here.</p>
    </v-flex-stub>
  </v-layout-stub>
</v-container-stub>
`;
